scripts/tslint-rule-to-eslint.js

```diff
@@ -0,0 +1,69 @@
+import { parse } from 'comment-json';
+import { promises as fs } from 'fs';
+import { format } from 'prettier';
+import * as path from 'path';
+
+const emptyObject = {};
+
+const parseAndReadFileContents = async filePath => {
+    try {
+        return parse((await fs.readFile(filePath)).toString());
```

Comment:
```ts
        const configFile =  await prettier.resolveConfigFile(filePath);
        const config  =  await prettier.resolveConfig(configFile);
        if (config) {
            config.editorconfig  = true;
        }
        return config;
```
IMMO that would use existing formatting setup (EditorConfig/Pretier) and remove the need for new dependency

Reply:
Nice, +1! Thanks!

Reply:
Just review in context of JSON5, as you mentioned in other place. Prettier supports only with 'json5' extension. Thanks!